fastlane/Fastfile

```diff
@@ -219,3 +308,24 @@ def replace_in(previous_text, new_text, path, allow_empty=false)
   backup_extension = '.bck'
   sh("sed", '-i', backup_extension, sed_regex, path)
 end
+
+def parse_previous_android_native_version
+  Dir.chdir(get_root_folder) do
+    return (sh 'cat android/build.gradle | grep "ext.purchases_version = " | awk \'{print($3)}\' | sed "s/\'//g"').strip
```

Comment:
maybe we can do this in ruby instead? 
it might be slightly longer but will definitely be much more readable

Reply:
I like this challenge. I am going to try haha because this is very ilegible

Reply:
Ok I think I did it haha [`0fa8d37` (#178)](https://github.com/RevenueCat/purchases-hybrid-common/pull/178/commits/0fa8d3749074a8af9c75dcb5770d1748f5a2386a)